Cargo.toml

```diff
@@ -10,9 +10,15 @@ name = "nom_locate"
 readme = "README.md"
 repository = "https://github.com/fflorent/nom_locate"
 version = "0.1.1"
+publish = false # breaking changes
 [badges.travis-ci]
 repository = "fflorent/nom_locate"
 
+[features]
+avx-accel = ["bytecount/avx-accel"]
+simd-accel = ["bytecount/simd-accel"]
+
 [dependencies]
+bytecount = "^0.3"
```

Comment:
Yup. I propose to [specify the git repository](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories) and merge the PR to a specific branch (say `release-0.2.0`).

Thus, we could start improving the benchmarks.

Reply:
I created the `release-0.2.0` branch